src/SFA.DAS.AANHub.Database/Tables/Partner.sql

```diff
@@ -1,10 +1,10 @@
 ﻿CREATE TABLE [dbo].[Partner]
 (
 	[MemberId] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY, 
-    [UKPRN] BIGINT NOT NULL, 
-    [Email] NVARCHAR(256) NULL, 
-    [Name] NVARCHAR(200) NULL, 
-    [LastUpdated] DATETIME NULL, 
+    [Organisation] NVARCHAR(200) NOT NULL,
```

Comment:
What is expected in the organisation field?

Reply:
It's the employer name, I believe this same field is in the employer request as well